src/vic3_world/buildings/building_importer.cpp

```diff
@@ -19,6 +19,12 @@ vic3::BuildingImporter::BuildingImporter()
    building_parser_.registerKeyword("goods_sales", [this](std::istream& input_stream) {
       goods_sales_value_ = static_cast<float>(commonItems::getDouble(input_stream));
    });
+   building_parser_.registerKeyword("staffing", [this](std::istream& input_stream) {
+      staffing_level_ = static_cast<float>(commonItems::getDouble(input_stream));
+   });
+   building_parser_.registerKeyword("production_methods", [this](std::istream& input_stream) {
+      production_methods_ = commonItems::getStrings(input_stream);
+   });
```

Comment:
Expand Vic3WorldBuildingsBuildingImporterTests::ItemsCanBeImported to cover this.

Reply:
Done.